debian/rules

```diff
@@ -52,12 +52,14 @@ DEPENDS += ansible, \
 	   cloud-init, \
 	   debootstrap, \
 	   debsums, \
+	   mtr, \
```

Comment:
I'd probably argue it'd be better to put these packages into the "human convenience" list later in this file.. I think we already have packages in that list that are actually required for the product, and thus not really "human convenience" (i.e. the comment is wrong).

But at the same time, I'm good with this.

I think it might be better to eventually combine the lists, since it's difficult to know which block to add new packages to, and there's no automatic enforcement (which leads to stale comments).

Reply:
I moved the packages to the correct location. Thanks for pointing this out.